Jenkinsfile

```diff
@@ -1,2 +1,8 @@
 /* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+buildPlugin(
```

Comment:
If you change this to `buildPlugin(jdkVersions: [11])` the ci will work. I've tested it in this draft PR: https://github.com/jenkinsci/prometheus-plugin/pull/480. I'll close the draft again after your PR is merged. No worries I won't steal your effort here - Just testing out things as I'm new here :)

Reply:
Thank you for your efforts

Reply:
Ok. Seems like it's not possible in that way: [https://docs.cloudbees.com/docs/cloudbees-ci-kb/latest/client-and-managed-masters/why-jenkinsfile-changes-are-not-reflected-in-pr-build](https://docs.cloudbees.com/docs/cloudbees-ci-kb/latest/client-and-managed-masters/why-jenkinsfile-changes-are-not-reflected-in-pr-build). If you don't mind I'll close this PR later today and merge my draft so that issue is resolved.

Reply:
OK, I understand, please help me close this issue and merge draft pr into main branch. 

Thx